_data/projects/OpinionatedGuides.yml

```
@@ -0,0 +1,11 @@
+name: Opinionated Guides
+desc: A one-stop guide to computers, linux, and electronics.
+site: https://github.com/opinionatedguide/opinionatedguide.github.io-src
+tags:
+- awesome
+- documentation
+- python
+- bash
+upforgrabs:
+  name: Hacktoberfest
+  link: https://opinionatedguide.github.io
```

This link was correct before:

```suggestion
  link: https://github.com/opinionatedguide/opinionatedguide.github.io-src/labels/Hacktoberfest
```

---

ahh, whoops, I changed link instead of site. Sorry, about that. Fixed